dotCMS/src/test/java/com/dotmarketing/image/focalpoint/FocalPointAPITest.java

```diff
@@ -0,0 +1,160 @@
+package com.dotmarketing.image.focalpoint;
+
+import static org.junit.Assert.assertTrue;
+import java.io.File;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.commons.io.FileUtils;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import com.dotcms.util.ConfigTestHelper;
+import com.dotmarketing.portlets.fileassets.business.FileAssetAPIImpl;
+import com.dotmarketing.util.UUIDGenerator;
+import com.google.common.collect.ImmutableMap;
+
+public class FocalPointAPITest {
+
+
+    final static String[] extensions = new String[] {"webp", "png", "gif", "jpg"};
+    static List<File> testFiles = new ArrayList<>();
+
+
+    static FocalPointAPIImpl fpAPI;
+
+
+
+    @BeforeClass
+    public static void setup() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4761083)